### PR TITLE
Add support for PVC labels

### PR DESCRIPTION
**Pull request by renetapopova**
To be cherry-picked when Helm Charts 5.10 are released.

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-919.surge.sh

File: modules/ROOT/pages/kubernetes/persistent-volumes.adoc [hunk 1]
```diff
@@ -158,6 +162,8 @@ When this method is used, the `persistentVolumeClaim` object must already exist.
 ----
 volumes:
   backups:
+  labels:
+        backup: true
     mode: volume
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/persistent-volumes.adoc:167]: here the mode is type volume so the label will not work

Comment (harshitsinghvi22) [reply modules/ROOT/pages/kubernetes/persistent-volumes.adoc:167]: we can remove the labels from here

Comment (renetapopova) [reply modules/ROOT/pages/kubernetes/persistent-volumes.adoc:167]: aaah, good catch!

File: modules/ROOT/pages/kubernetes/persistent-volumes.adoc [hunk 2]
```diff
@@ -82,6 +83,8 @@ neo4j:
   name: standalone-with-storage-class
 volumes:
   data:
+    labels:
+        data: true
```
Comment (harshitsinghvi22) [inline modules/ROOT/pages/kubernetes/persistent-volumes.adoc:87]: can we also quote the values "true" ...i believe without quotes the example might just fail...so better to quote the value in documentation

Comment (renetapopova) [reply modules/ROOT/pages/kubernetes/persistent-volumes.adoc:87]: But that's a boolean. Are you sure?

Comment (harshitsinghvi22) [reply modules/ROOT/pages/kubernetes/persistent-volumes.adoc:87]: yeah since its a label...labels are considered strings

Comment (renetapopova) [reply modules/ROOT/pages/kubernetes/persistent-volumes.adoc:87]: aah, ok.

**Review by harshitsinghvi22 — APPROVED** [review]

**Pull request merged**